brave/brave-core
# Remove Restore tab from Manage Brave Rewards dialog

Resolves https://github.com/brave/brave-browser/issues/26338

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on
  
## UI:

![Screenshot 2022-11-01 105717](https://user-images.githubusercontent.com/5033691/199266431-1bc2b10d-05f4-4ca8-bfd8-b8199dff24db.png)

## Test Plan:

- Clean profile
- Enable Rewards
- Visit brave://rewards
- Click "Manage Brave Rewards" button
- Verify that Manage Brave Rewards dialog only has two tabs: Backup and Reset
